### ISSUE TITLE
Report list: Number of entries and Pagination

### ISSUE THREAD

**Issue by ghost** [idmc-labs/idmc-monitoring-platform-feedback#4]
Should there be a number to show how many entries per dataset? 
![screen shot 2018-03-05 at 10 57 21 am](https://user-images.githubusercontent.com/1164551/36993847-192572a4-2064-11e8-821d-d69aff7b5205.png)

Pagination shows there are more pages after searching filtered list but there are no additional entries after page 1
![screen shot 2018-03-05 at 10 56 39 am](https://user-images.githubusercontent.com/1164551/36993866-31b029c2-2064-11e8-8b16-7880e1021aa6.png)

**Comment by milanoleonardo** [comment]
Good catch! I know... I did it like this in order to speed up the loading. Basically the total number of entries is "estimated" by Postgres rather than calculated. I will see what I can do to fix it
